rasa/core/policies/mapping_policy.py

```diff
@@ -59,15 +59,24 @@ def predict_action_probabilities(self,
             elif intent == USER_INTENT_BACK:
                 idx = domain.index_for_action(ACTION_BACK_NAME)
                 prediction[idx] = 1
+            if any(prediction):
+                logger.debug("The predicted intent is '{}', which is mapped to "
+                             " action '{}' in the domain."
+                             "".format(intent, action))
         elif tracker.latest_action_name == action and action is not None:
             latest_action = tracker.get_last_event_for(ActionExecuted)
             assert latest_action.name == action
-
+            logger.debug("The mapped action, '{}', for this intent, '{}', was "
+                         "executed last so MappingPolicy is returning to "
+                         "action_listen.".format(action, intent))
```

Comment:
Should this message go into the `if` logic in line 72? It'll only return to `action_listen` if we get inside there, right?